.idea/codeStyles/Project.xml

```diff
@@ -55,7 +55,7 @@
         </value>
       </option>
       <option name="NAME_COUNT_TO_USE_STAR_IMPORT" value="2147483647" />
-      <option name="NAME_COUNT_TO_USE_STAR_IMPORT_FOR_MEMBERS" value="1" />
+      <option name="NAME_COUNT_TO_USE_STAR_IMPORT_FOR_MEMBERS" value="2147483647" />
```

Comment:
This means "don't use star imports" — KtLint and detekt really hate star imports